compose/cli/main.py

```diff
@@ -308,13 +308,24 @@ def logs(self, project, options):
         Usage: logs [options] [SERVICE...]
 
         Options:
-            --no-color  Produce monochrome output.
+            --no-color          Produce monochrome output.
+            -f, --follow        Follow log output.
+            -t, --timestamps    Show timestamps.
+            --tail="all"        Number of lines to show from the end of the logs
+                                for each container.
         """
         containers = project.containers(service_names=options['SERVICE'], stopped=True)
 
         monochrome = options['--no-color']
+        log_args = {}
+        log_args['stream'] = options['--follow']
```

Comment:
`stream` and `follow` should be different things, however it looks like they were set to the same thing in `docker-py`. `stream` should say "return an iterator", where as follow is a behaviour of the API to continue to send logs or stop once it receives the latest one.

I think we need to fix `docker-py` to have separate params, then this should be:

```
logs_args['follow'] = ...
```

`stream` should always be `True`.